### ISSUE TITLE
When opening outlook, the compose message dialog does not open

### ISSUE THREAD

**Issue by mrabomination** [Noitidart/MailtoWebmails#42]
Exactly as the title says; whenever outlook.com is launched the compose new message does not open, instead it just sits on my inbox.

**Comment by Noitidart** [comment]
Yeah this isue happend and I emailed their support team, but they haven't got back to me. I might have to just dive in and try to find out the new template url.

**Comment by mrabomination** [comment]
I'm pretty sure that's what the root of the issue is, even just typing the URL that should open a new message doesn't work.

**Comment by Noitidart** [comment]
I just updated the URL, please load about:mailto and it will update it. It is now opening the compose page, however it is not stripping the prefix of `mailto:`, I asked the support team but they are not getting back to me, so I created a public discussion topic:

http://answers.microsoft.com/en-us/outlook_com/forum/oemail-ocompose/mailto-handler/a0d79106-2819-4410-ac6d-19d1d2d683b4?tm=1470852303152

**Comment by anyhowes** [comment]
Hi There,

I just got this add-on, noticed this problem, and went to your GitHub issues page which lead me to this thread. Any updates? Normally it wouldn't be so bothersome since you could manually delete the "mailto:" portion of the link, but because of how the new Outlook.com displays email addresses, it involves a few more clicks to manually edit addresses.

**Comment by Noitidart** [comment]
Hi @anyhowes - I have not looked into stripping the `mailto:` prefix yet. I plan to rework the internals of the addon and hope to add a default "strip `mailto:` prefix if it wasn't" feature. Maybe we should file a separate issue topic dedicated to that huh? What do you think?

**Comment by anyhowes** [comment]
I think that's a swell idea. Not the highest priority, but it will keep it on your radar.
Thanks!

**Comment by Grodge** [comment]
Hi,

This used to work perfectly, but since the new changes to Outlook.com, the inbox is presented, but nothing else. I am basically reiterating what this thread is about already. Any chance of a fix? It is otherwise a great extension, and I'm now not sure whether to remove it, as the weeks are rolling by with no fixes. thanks so much! Sophie

**Comment by Noitidart** [comment]
@Grodge thanks for showing your vote. I badly want to fix this, I even reached out to Microsoft team but they aren't responding to me.

The community can help me get a fix for this though, please try searching the web and email Microsoft asking them for this. I have done this but but hopefully the community will have more luck. :)

**Comment by Grodge** [comment]
Hi Noitidart,

Thank you so much for your reply!  Yes, I realised that you had this issue with MS not responding, and I also could see your great wish to put this right. It is a unique extension, as far as I know, for turning mailto: links into webmail-enabled, and I really hope you can make some progress with it.

In the meantime, thank you for all your help to the community.

Kind regards,

George

**Comment by Grodge** [comment]
Microsoft give such "generic" answers, that it seems almost impossible to get something sensible from them. I will try and make contact though, thanks!

**Comment by Noitidart** [comment]
@Grodge I appreciate your comments very much! :)

I had this problem with Google Inbox as well, I worked with the community on it but eventually a hero figured it out and he saved us all the headache of communicating with Google. I'm hoping as a community we can knock out this one as well! It's what it takes with corporate giants like these and their poor support teams for non-paying clients.

**Comment by Grodge** [comment]
Hi again, and thank you for your reply. Sadly I'm not a coder, but many times community-based approach is the "only" way, because large corporations like Google and MS are practically unreachable, very slow to put things right, and very generic in their responses if they respond at all.

I have seen their replies on forums, and they literally cut and paste "pre made" answers, and don't even appear to read what the thread is about in the first place.

They also change things all the time, so it is a game of cat and mouse trying to keep up with what they have done. Nothing stays the same for long!

I love your extension, and I hope between any/all interested parties, we can keep up with that cat and mouse game that MS give us!

Best wishes,

George

**Comment by fankmd** [comment]
I am using a macbook.  When I right click on the link, it does not give me the option to open in a new tab (see screenshot below).  When I click it opens my Yahoo mail in the same tab, initially it opens the inbox page, but if I wait about 45 seconds, the compose page comes up properly.

![image](https://cloud.githubusercontent.com/assets/25400520/22395723/e46c58a0-e53d-11e6-9c72-6f931b44660b.png)
